tests/test_configfile.py

```diff
@@ -211,3 +213,9 @@ def test_config_file_dash_replace(self, input):
             del os.environ['OELINT_CONFIG']
 
             assert getattr(_args, _option.replace('-', '_'))
+
+
+    def test_option_deserialization(self):
+        options = {'a': 'True', 'b': True, 'c': 'False', 'd': False, 'e': 'other value'}
+        deserialized = deserialize_boolean_options(options)
+        assert deserialized == {'a': True, 'b': True, 'c': False, 'd': False, 'e': 'other value'}
```

Comment:
I see your point in this change.
What I'm not sure about is how this interacts with argparse

how about one of the builtin options? like

```python
parser.add_argument('--nobackup', action='store_true', default=False, help='Don\'t create backup file when auto fixing')
```
Could you please add a testcase that tests that `nobackup=False` in the config file still ends up in nobackup=False in the argparse namespace

Reply:
Sorry my bad - just had a look at the code that is calling this function - it's should work perfectly as proposed

Reply:
I'm not sure I understand. The booleans are all treated the same, no matter if it's applied to `--nobackup` or e.g. `--quiet`. The previous logic was incorrect apparently as `False` provided as a `str` to a `default` with `action=store_true` was `True`.